package.json

```diff
@@ -42,6 +43,7 @@
     "hof-middleware-markdown": "^1.0.0",
     "hof-model": "^3.0.0",
     "hof-transpiler": "^0.2.1",
+    "html-pdf": "^2.1.0",
```

Comment:
This creates a dependency on `phantomjs-prebuilt` which might cause problems with our policy of not allowing install scripts to run in containers, because it has one.